crates/runtime/src/runtime.rs

```diff
@@ -135,6 +135,9 @@ impl RuntimeBuilder {
         #[allow(clippy::cast_possible_truncation)]
         pooling_config
             .total_component_instances(self.max_components)
+            .total_core_instances(self.max_components)
+            .total_gc_heaps(self.max_components)
+            .total_stacks(self.max_components)
```

Comment:
So, put another way, each component instance gets one core instance, one GC heap, and one stack?

Reply:
Yes that is correct